legacy/generate_checkpoints.sh

```diff
@@ -22,7 +23,7 @@ do
   # activate and install PL version
   source "$ENV_PATH/bin/activate"
   # there are problem to load ckpt in older versions since they are saved the newer versions
-  pip install "pytorch_lightning==$ver" "torch==1.3" --quiet --no-cache-dir
```

Comment:
Awesome, the wheels for this Pytorch version was not available for the latest Python versions